src/vcacheoptimizer.cpp

```diff
@@ -268,11 +264,6 @@ void meshopt_optimizeVertexCacheTable(unsigned int* destination, const unsigned
 		cache = cache_new, cache_new = cache_temp;
 		cache_count = cache_write > cache_size ? cache_size : cache_write;
 
-		// update live triangle counts
```

Comment:
This should note that live triangle counts are updated as a byproduct of adjacency adjustment below